lib/LocalSigner.ts

```diff
@@ -0,0 +1,26 @@
+import ISigner from './interfaces/ISigner';
+import InputValidator from './InputValidator';
+import JwkEs256k from './models/JwkEs256k';
+import OperationKeyType from './enums/OperationKeyType';
+const secp256k1 = require('@transmute/did-key-secp256k1');
+
+/**
+ * An ISigner implementation that uses a given local private key.
+ */
+export default class LocalSigner implements ISigner {
+  /**
+   * Creates a new local signer using the given private key.
+   */
+  public static create (privateKey: JwkEs256k): ISigner {
```

Comment:
Why have a static create and a constructor?

Reply:
This is the "static factory method" design pattern. You can read upon why it might be good, first link found in search: https://stackoverflow.com/questions/929021/what-are-static-factory-methods.

Key take away is that it is in general a good practice. Bottom line is that it hides details on what/how an instance is created, as the caller doesn't really need to know, notice I return `ISigner` not `LocalSigner`.